.github/workflows/pkgci_regression_test_cpu.yml

```diff
@@ -58,20 +58,36 @@ jobs:
             experimental/regression_suite
 
       # Out of tree tests
-      - name: Checking out external TestSuite repository
+      - name: Install git lfs
+        run: |
+          sudo apt-get install git-lfs
+          git lfs install
+      - name: Check out external TestSuite repository
         uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
         with:
           repository: nod-ai/SHARK-TestSuite
-          ref: 46ab63b271fddd78e071c0590e9a073f6282b005
+          ref: 65cdac8e1dfab53892337009a9704162e1cf8bfb
           path: SHARK-TestSuite
           submodules: false
-      - name: Installing external TestSuite Python requirements
+          lfs: true
+      - name: Install external TestSuite Python requirements
         run: |
           source ${VENV_DIR}/bin/activate
           python -m pip install -r SHARK-TestSuite/iree_tests/requirements.txt
-      - name: Run external TestSuite tests
-        env:
-          IREE_TEST_CONFIG_FILES: experimental/regression_suite/external_test_suite/config_cpu_llvm_sync.json
+      - name: Run external tests - ONNX test suite
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest SHARK-TestSuite/iree_tests/onnx/ \
+              -n auto -rpfE --timeout=30 --retries=2 --durations=20 \
+              --config-files=experimental/regression_suite/external_test_suite/onnx_cpu_llvm_sync.json
+      # TODO(scotttodd): add a local cache for these large files to a persistent runner
+      - name: Download remote files for real weight model tests
+        run: |
+          source ${VENV_DIR}/bin/activate
+          python SHARK-TestSuite/iree_tests/download_remote_files.py
+      - name: Run external tests - Models with real weights
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest SHARK-TestSuite/iree_tests -n auto -rpfE --timeout=30 --retries=2
+          pytest SHARK-TestSuite/iree_tests/pytorch/ -k real_weights \
+               -rpfE --timeout=600 --durations=0 \
+              --config-files=experimental/regression_suite/external_test_suite/pytorch_cpu_llvm_task.json
```

Comment:
As we add more programs to the test suite, we can do other filtering here, using either the directory (e.g. `iree_tests/pytorch/`), test names (e.g. `-k real_weights`), or other markers. We can filter both the remote file downloading and the test running. We can also shard the test running across multiple workers (given enough runners in the repo/org)

We can also have separate config files for each slice of the test suite if we want to adjust flags for each test case or run a single test case with multiple combinations of flags.